### PR TITLE
Create doctor institution window

### PR DESCRIPTION
**Pull request by VictorAgovski**

### CODE REVIEW

File: lib/screens/bottom_nav_bar/bottom_nav_bar.dart [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_gp5/extensions/build_context_extensions.dart';
+import 'package:iconly/iconly.dart';
+import '../../locale/l10n/app_locale.dart';
+import '../../repos/authentication/authentication_repository.dart';
+import '../../routes/app_routes.dart';
+
+class BottomNavigationBarWidget extends StatelessWidget {
```
Comment (HristoPetkov94) [inline lib/screens/bottom_nav_bar/bottom_nav_bar.dart:9]: this should be in widgets package

**Comment by HristoPetkov94** [comment]
when selecting doctor, the loading indicator should be in the middle of the page

**Comment by HristoPetkov94** [comment]
categories should be added with foldable option as it is in the design

**Comment by HristoPetkov94** [comment]
the sizes seems to be not the same
image

**Comment by VictorAgovski** [comment]
> the sizes seems to be not the same image

The size is the same, just there is no fields about rating and city and the each doctor has only one specialization (if there were several specializations the design would look the same)

**Review by HristoPetkov94 — APPROVED** [review]

**Pull request merged**